users/toomanydaves/website/index.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html lang="en-US">
+    <head>
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width,initial-scale=1">
+        <title>Sean Paul Jacke</title>
+        <script src="js/main.js"></script>
```

Comment:
I'd recommend using relative path instead of absolute path

Reply:
@CringleySDays I appreciate the help, but I'm pretty sure these are relative paths. They're just relative to the current directory, as opposed to relative to the root, which is what you get when the path starts with /. An absolute path would look like http://some-website.com/resource

Reply:
Oh yeah they are, I must have mis-read them, my bad! I thought I saw `/js/main.js` which is absolute path